src/components/SearchBar.js

```diff
@@ -1,13 +1,19 @@
 import React from 'react';
 
-const SearchBar = ({ searchField, searchChange }) =>
+const SearchBar = ({ search, handleSearchChange, handleSubmit }) =>
    (
     <div className="pa2">
       <input
         className="bw2 pa2 b--blue w-two-thirds-ns"
         type="search"
-        placeholder="Search (filter) Movies!"
-        onChange={searchChange}
+        placeholder="Search..."
+        onChange={handleSearchChange}
+      />
+      <input
+        disabled={!search}
+        className="pa2 bw2 b--blue"
+        type="submit"
+        onClick={handleSubmit}
       />
```

Comment:
What about adding a Clear button that resets the search query and results?

Reply:
done!